docs/docs/guides/integrate/identity-providers/introduction.md

```diff
@@ -0,0 +1,182 @@
+- [LDAP](./ldap) 
+- [Local OpenLDAP](./openldap.mdx)
+- [OKTA generic OIDC](./okta-oidc)
+- [OKTA SAML](./okta-saml)
+- [Keycloak generic OIDC](./keycloak)
+- [MockSAML](./mocksaml)
+- [JWT IdP](./jwt_idp)
+
+
+### Configuring IdPs without predefined templates
+
+If ZITADEL doesn't offer a specific template for your Identity Provider (IdP) and your IdP is fully compliant with OpenID Connect (OIDC), you have the option to use the generic OIDC provider configuration.
+
+For those utilizing a SAML Service Provider, the SAML Service Provider option is available. You can learn how to set up a SAML Service Provider with our [MockSAML example](https://zitadel.com/docs/guides/integrate/identity-providers/mocksaml).
+
+Should you wish to transition from a generic OIDC provider to Entra ID (formerly Azure Active Directory) or Google, consider following this [guide](https://zitadel.com/docs/guides/integrate/identity-providers/migrate).
+
+
+
+## Key settings on the templates
+
+When configuring external IdP templates in ZITADEL, several common settings enable customized integration to suit your application's authentication flow. Below is a generic explanation of these settings:
+
+- **Scopes**: Specifies the permissions your application requests from the user's account on the external IdP. Common scopes include `openid`, `profile`, and `email`, essential for accessing basic user information for authentication and account management in ZITADEL. You can specify additional scopes based on your application's requirements and the information needed from the external IdP.
+
+- **Automatic creation**: When enabled, this allows ZITADEL to automatically create a new user account if someone logs in with their external IdP credentials and no corresponding account exists in ZITADEL. This facilitates a smooth user onboarding experience by eliminating the need for manual account creation.
+
+- **Automatic update**: This feature, when activated, allows ZITADEL to automatically update a user's profile information whenever changes are detected in the user's account on the external IdP. For example, if a user changes their last name in their Google or Microsoft account, ZITADEL will reflect this update in the user's account upon their next login.
+
+- **Account creation allowed**: Determines whether new user accounts can be created in ZITADEL through the external IdP authentication process. Enabling this setting is crucial for allowing users who are new to your application to register and create accounts seamlessly via their existing external IdP accounts.
+
+- **Account linking allowed**: Enables existing ZITADEL accounts to be linked with identities from external IdPs. It requires that a linkable ZITADEL account already exists for the user attempting to log in with an external IdP. Account linking is beneficial for users who wish to associate multiple login methods with their ZITADEL account, providing flexibility and convenience in how they access your application.
```

Comment:
It might be worth putting this in a separat file and import here.
We could then also include that in all the identity provider guides. We already have that there, but with a shorter description.
Would be nice to have it only once.

Reply:
Fabi, I thought about this and I think this longer description is suitable for the intro page and we can keep it short for the separate pages which include the text in one place anyway.